packages/fiori-freestyle/test/basic.test.ts

```diff
@@ -55,16 +55,22 @@ describe(`Fiori freestyle template: ${TEST_NAME}`, () => {
     ];
 
     beforeAll(() => {
-        rmdirSync(curTestOutPath, { recursive: true });
+        removeSync(curTestOutPath);  // even for in memory
     });
 
     test.each(configuration)('Generate files for template: $name', async ({ name, config }) => {
         const testPath = join(curTestOutPath, name);
         const fs = await generate(testPath, config);
         expect((fs as any).dump(testPath)).toMatchSnapshot();
-        // write out the files for debugging
+
         return new Promise((resolve) => {
-            fs.commit(resolve);
+					// write out the files for debugging
+					if (debug?.enabled) {
+						fs.commit(resolve);
+					} else
+					{
```

Comment:
```suggestion
					} else {
```
Minor change to maintain consistency with other formatting.